src/lib/flocking/ugens/envelopes.js

```diff
@@ -0,0 +1,1101 @@
+/*
```

Comment:
I will be removing Bower support from Flocking 0.3.0, so I recommend sourcing it from npm. Since there has been a long time between Flocking 0.1.0 and the as-yet-unreleased 0.2.0 version, I recommend using the latest commit hash. So the package.json dependency would look something like this:

    "flocking": "colinbdclark/flocking#fba0738886b3b6aa1f6292863d2fa50a39cd47e3"

In the future, I will investigate using Fluid's npm publishing tools to more frequently publish development snapshots.